subprojects/platform-play/src/main/java/org/gradle/play/internal/run/PlayRunWorkerServerProtocol.java

```diff
@@ -21,5 +21,5 @@
 public interface PlayRunWorkerServerProtocol extends Stoppable {
     void buildSuccess();
     void buildError(Throwable throwable);
-    void rebuildInProgress();
+    void expectPendingChanges();
```

Comment:
I don't really understand this: who is expecting pending changes? Is this for Gradle to tell the Play app "I'm running now, wait for the build to complete.